src/opencost_parquet_exporter.py

```diff
@@ -1,14 +1,31 @@
-"""OpenCost parquet exporter.
+# pylint: disable=W0511
 
-This module exports data from OpenCost API to parquet format, making it suitable
-for further analysis or storage in data warehouses.
 """
+This module provides an implementation of the OpenCost storage exporter.
+"""
+
 import sys
 from datetime import datetime, timedelta
 import os
+import json
 import pandas as pd
 import requests
-import botocore.exceptions as boto_exceptions
+from storage_factory import get_storage
+
+
+def load_config_file(file_path: str):
```

Comment:
add some test cases related to the data types, ignore_alloc_keys and rename_columns